curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/6443c98ac169e501eba36d04.md

```diff
@@ -0,0 +1,288 @@
+---
+id: 6443c98ac169e501eba36d04
+title: Step 28
+challengeType: 0
+dashedName: step-28
+---
+
+# --description--
+
+Create a constant named `MS_PER_MINUTE` that will store the number of milliseconds in a minute. There are `1000 * 60` milliseconds in a minute. You can do the calculation like this :
+
+```js
+const MS_PER_MINUTE = 1000 * 60;
+```
```

Comment:
But also I am not a huge fan that we are back to using the screaming snake case convention here for this one variable. 

To me, this opens up the door of confusion for beginners seeing this mixture of casing conventions.

<img width="666" alt="Screenshot 2023-11-21 at 10 55 01 AM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/3c021488-6877-4278-8b18-e4db5f911e1d">

I would like to hear @naomi-lgbt and @scissorsneedfoodtoo 's thoughts on this.

Should we change this variable to be `camelCase`?

<img width="681" alt="Screenshot 2023-11-21 at 10 56 39 AM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/92a2933b-2778-4aec-b19f-c1a991632c0e">

Or should we just remove this step all together and add this `const` variable with the screaming snake case convention with the other ones we added at the beginning here?

<img width="843" alt="Screenshot 2023-11-21 at 11 03 54 AM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/5d606792-8ac4-4ace-a433-0965ae0fc959">

Reply:
at the moment i am following this : https://github.com/freeCodeCamp/CurriculumExpansion/blob/main/javascript-algorithms-and-data-structures/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/script.js


if you think we should change, open a new comment below so that i can start implementation asap

Reply:
We decided in the last JS meeting that all instances of screaming snake case will be changed camel case for consistency. 

For this step, `MS_PER_MINUTE` is now changed to `msPerMinute`.
Please also update all of the seed code for the remaining steps too

Reply:
added